packages/chrome.rb

```diff
@@ -21,21 +26,17 @@ class Chrome < Package
     depends_on 'sommelier'
   end
 
-  def self.build
-    Dir.chdir '..' do
-      system "alien -tc #{@_deb}"
-      system "tar xf google-chrome-stable-#{@_ver}.tgz"
-    end
-  end
-
   def self.install
-    Dir.chdir '..' do
-      ENV['CREW_NOT_STRIP'] = '1'
-      FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
-      FileUtils.mv 'opt/', CREW_DEST_PREFIX
-      FileUtils.mv 'usr/share', CREW_DEST_PREFIX
-      FileUtils.mv 'etc', CREW_DEST_PREFIX
-      FileUtils.ln_s "#{CREW_PREFIX}/opt/google/chrome/google-chrome", "#{CREW_DEST_PREFIX}/bin/google-chrome"
-    end
+    ENV['CREW_NOT_STRIP'] = '1'
+    FileUtils.mkdir_p CREW_DEST_PREFIX
+    FileUtils.mkdir_p 'bin'
+    
+    FileUtils.mv 'usr/share', 'share'
+    FileUtils.rm_rf 'usr'
+    
+    FileUtils.ln_s "#{CREW_PREFIX}/opt/google/chrome/google-chrome", 'bin/google-chrome-stable'
+    FileUtils.ln_s "#{CREW_PREFIX}/opt/google/chrome/google-chrome", 'bin/google-chrome'
```

Comment:
How about `FileUtils.mv 'opt/google/chrome/google-chrome', 'share'` and then:
```
FileUtils.ln_s "#{CREW_PREFIX}/share/google-chrome", 'bin/google-chrome-stable'
FileUtils.ln_s "#{CREW_PREFIX}/share/google-chrome", 'bin/google-chrome'
```
This is a better location than opt.

Reply:
I think `opt` is the best location for chromium-based browsers as programs there usually live inside a single folder that contains all of their data, libraries, etc.

Reply:
I think `share` is better than `opt` for consistency with other packages.  I don't want package files stored all over the place in different locations.  Wouldn't it be nice if you could cd to the share directory and list all the installed apps in one location?

Reply:
Fair enough :)

Reply:
Change made